packages/cli-repl/src/cli-repl.spec.ts

```diff
@@ -0,0 +1,17 @@
+import mapCliToDriver from './arg-mapper';
+import parseCliArgs from './arg-parser';
+import CliRepl from './cli-repl';
+import { expect } from 'chai';
+
+describe('cli-repl', () => {
+  context('constructor', () => {
+    it('starts without errors', (done) => {
+      const options = parseCliArgs([]);
+      const driverOptions = mapCliToDriver(options);
+      const cli = new CliRepl('mongodb://127.0.0.1:27017', driverOptions, options)
```

Comment:
since `new CliRepl()` starts node repl the stdin and stdout are messed up entirely.

An option there is to pass mocks of those streams.